utils/fixed_decimal/src/decimal.rs

```diff
@@ -948,6 +956,14 @@ impl FixedDecimal {
     /// )
     /// .expect("Finite, integer-valued quantity");
     /// assert_eq!(decimal.write_to_string(), "12345678000");
+    ///
+    /// // IEEE 754 for floating point defines the sign bit separate
```

Comment:
Nit: Make these actually separate examples

```suggestion
    /// assert_eq!(decimal.write_to_string(), "12345678000");
    /// ```
    ///
    /// Negative zero is supported:
    ///
    /// ```
    /// // IEEE 754 for floating point defines the sign bit separate
```